nomad/resource_acl_auth_method.go

```diff
@@ -264,21 +279,54 @@ func resourceACLAuthMethodRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	log.Printf("[DEBUG] Reading ACL Auth Method %q", authMethodName)
-	authMethod, _, err := client.ACLAuthMethods().Get(authMethodName, nil)
+	authMethod, err := fetchResource(client, authMethodName)
 	if err != nil {
 		return fmt.Errorf("error reading ACL Auth Method %q: %s", authMethodName, err.Error())
 	}
 	log.Printf("[DEBUG] Read ACL Auth Method %q", authMethod.Name)
 
+	unredactResource(d, authMethod, nil)
+	setStateFromResource(d, authMethod)
+	return nil
+}
+
+func fetchResource(client *api.Client, name string) (*api.ACLAuthMethod, error) {
```

Comment:
the package here is `nomad` (the whole provider) so I think this method is named too broadly for its namespace. ditto the other funcs below.

Reply:
Oof, totally right. I was getting file tunnel vision